Packages/com.unity.inputsystem/InputSystem/API/Input.cs

```diff
@@ -731,28 +731,48 @@ public static bool IsControlUp(Inputs input)
             }
         }
 
-        // RE-ENABLE ME WHEN YOU GONNA IMPLEMENT ME
-#if false
         /// <summary>
-        /// For single analogue inputs
+        /// Returns actuation value for single analogous controls.
         /// </summary>
-        /// <param name="input"></param>
+        /// <param name="input">Control from Inputs enum.</param>
         /// <returns>A value between 0 at un-actuated to 1 at fully actuated.</returns>
-        /// <exception cref="NotImplementedException"></exception>
         public static float GetAxis(Inputs input)
         {
-            throw new NotImplementedException();
+            var deviceType = GetDeviceTypeForInput(input);
+
+            switch (deviceType)
+            {
+                case InputDeviceType.Keyboard:
+                    return InputSystem.devices
+                        .OfType<Keyboard>()
+                        .Max(x => Mathf.Clamp01(GetKeyboardButtonControl(x, input).ReadValue()));
```

Comment:
All of these Max lambdas will allocate on every call. Think we might have to do it the old fashioned way